erpnext/accounts/doctype/payment_reconciliation/payment_reconciliation.py

```diff
@@ -89,11 +89,10 @@ def get_unreconciled_entries(self):
 	def get_nonreconciled_payment_entries(self):
 		self.check_mandatory_to_fetch()
 
-		payment_entries = self.get_payment_entries()
-		journal_entries = self.get_jv_entries()
-
+		payment_entries = self.get_payment_entries() or []
+		journal_entries = self.get_jv_entries() or []
 		if self.party_type in ["Customer", "Supplier"]:
-			dr_or_cr_notes = self.get_dr_or_cr_notes()
+			dr_or_cr_notes = self.get_dr_or_cr_notes() or []
```

Comment:
all these functions already return a list, always. what is the reason for making this change?